samples/grids/data-grid/type-marketing-table/wwwroot/DataGridBindingLiveData.js

```diff
@@ -0,0 +1,407 @@
+// NOTE this JavaScript file implements functions for styling/templating columnd of the DataGrid control
+// at end of this file, the igRegisterScript registers functions that are used .razor file
+
```

Comment:
this JS  file needs to match name of razor file, e.g
DataGridTypeMarketingTable.js
also update index.html after renaming this JS file